tours/constants.py

```diff
@@ -23,13 +21,19 @@
     12: 4,
 }
 
-GROUP_NAME = {
```

Comment:
These mappings document the code quite nicely, at least for casual reader. Are they removed in this PR? Is it still possible to name groups somehow..?

Reply:
They are removed, because they are not used in anywhere in the code. I agree with readability though. I will do something to this when the groups have been verified to work in estimation. Until that they might change still.